src/test/java/com/googlecode/lanterna/TestACS.java

```diff
@@ -27,66 +29,14 @@
-        System.out.println("DOUBLE_LINE_VERTICAL = " + Symbols.DOUBLE_LINE_VERTICAL);
-        System.out.println();
-        System.out.println("SINGLE_LINE_TOP_LEFT_CORNER = " + Symbols.SINGLE_LINE_TOP_LEFT_CORNER);
-        System.out.println("DOUBLE_LINE_TOP_LEFT_CORNER = " + Symbols.DOUBLE_LINE_TOP_LEFT_CORNER);
-        System.out.println("SINGLE_LINE_TOP_RIGHT_CORNER = " + Symbols.SINGLE_LINE_TOP_RIGHT_CORNER);
-        System.out.println("DOUBLE_LINE_TOP_RIGHT_CORNER = " + Symbols.DOUBLE_LINE_TOP_RIGHT_CORNER);
-        System.out.println();
-        System.out.println("SINGLE_LINE_BOTTOM_LEFT_CORNER = " + Symbols.SINGLE_LINE_BOTTOM_LEFT_CORNER);
-        System.out.println("DOUBLE_LINE_BOTTOM_LEFT_CORNER = " + Symbols.DOUBLE_LINE_BOTTOM_LEFT_CORNER);
-        System.out.println("SINGLE_LINE_BOTTOM_RIGHT_CORNER = " + Symbols.SINGLE_LINE_BOTTOM_RIGHT_CORNER);
-        System.out.println("DOUBLE_LINE_BOTTOM_RIGHT_CORNER = " + Symbols.DOUBLE_LINE_BOTTOM_RIGHT_CORNER);
-        System.out.println();
-        System.out.println("SINGLE_LINE_CROSS = " + Symbols.SINGLE_LINE_CROSS);
-        System.out.println("DOUBLE_LINE_CROSS = " + Symbols.DOUBLE_LINE_CROSS);
-        System.out.println();
-        System.out.println("SINGLE_LINE_T_UP = " + Symbols.SINGLE_LINE_T_UP);
-        System.out.println("SINGLE_LINE_T_DOWN = " + Symbols.SINGLE_LINE_T_DOWN);
-        System.out.println("SINGLE_LINE_T_RIGHT = " + Symbols.SINGLE_LINE_T_RIGHT);
-        System.out.println("SINGLE_LINE_T_LEFT = " + Symbols.SINGLE_LINE_T_LEFT);
-        System.out.println();
-        System.out.println("SINGLE_LINE_T_DOUBLE_UP = " + Symbols.SINGLE_LINE_T_DOUBLE_UP);
-        System.out.println("SINGLE_LINE_T_DOUBLE_DOWN = " + Symbols.SINGLE_LINE_T_DOUBLE_DOWN);
-        System.out.println("SINGLE_LINE_T_DOUBLE_RIGHT = " + Symbols.SINGLE_LINE_T_DOUBLE_RIGHT);
-        System.out.println("SINGLE_LINE_T_DOUBLE_LEFT = " + Symbols.SINGLE_LINE_T_DOUBLE_LEFT);
-        System.out.println();
-        System.out.println("DOUBLE_LINE_T_UP = " + Symbols.DOUBLE_LINE_T_UP);
-        System.out.println("DOUBLE_LINE_T_DOWN = " + Symbols.DOUBLE_LINE_T_DOWN);
-        System.out.println("DOUBLE_LINE_T_RIGHT = " + Symbols.DOUBLE_LINE_T_RIGHT);
-        System.out.println("DOUBLE_LINE_T_LEFT = " + Symbols.DOUBLE_LINE_T_LEFT);
-        System.out.println();
-        System.out.println("DOUBLE_LINE_T_SINGLE_UP = " + Symbols.DOUBLE_LINE_T_SINGLE_UP);
-        System.out.println("DOUBLE_LINE_T_SINGLE_DOWN = " + Symbols.DOUBLE_LINE_T_SINGLE_DOWN);
-        System.out.println("DOUBLE_LINE_T_SINGLE_RIGHT = " + Symbols.DOUBLE_LINE_T_SINGLE_RIGHT);
-        System.out.println("DOUBLE_LINE_T_SINGLE_LEFT = " + Symbols.DOUBLE_LINE_T_SINGLE_LEFT);
+    {
+        for(Field field : Symbols.class.getFields()) {
+            field.setAccessible(true);
+            try {
+                System.out.printf("%1$s = %2$s%n", field.getName(), field.get(null));
```

Comment:
This has the disadvantage of losing the grouping and formatting above. We could keep a list of groups to print and the use reflections to print anything that wasn't in the list.